### DISCUSSION TITLE
Is there a way to Keyframe/Tween a layer to transform with Perspective?

### DISCUSSION THREAD

**Discussion by FarKnyte** [keiwando/toonsquid#377] [General]
I’m trying to tween/keyframe this layer but it doesn’t seem to like it when I choose perspective to transform it

It work fine with Uniform and Freeform but not Perceptive or Warp?

Is this a limit or I’m I just stupid?

**Comment by VeiloPot** [comment]
> I’m trying to tween/keyframe this layer but it doesn’t seem to like it when I choose perspective to transform it
> 
> 
> 
> It work fine with Uniform and Freeform but not Perceptive or Warp?
> 
> 
> 
> Is this a limit or I’m I just stupid?

It was mentioned at one point that it would be a feature that would be released in a future update.
